### PR TITLE
Created examples for switch statements.

### PR DESCRIPTION
**Pull request by shivanishirolkar**
Hi, I added an example for switch statements. Thanks!

### CODE REVIEW

File: SwitchStatements/src/season/Season.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	 *            Month
+	 * @return the season
+	 */
+	public static String determineSeason(String month) {
+		String season = null;
+		switch (month) {
+		case "November":
+		case "December":
+		case "January":
+		case "February":
```
Comment (xaca) [inline SwitchStatements/src/season/Season.java:44]: season = month.toLowerCase() and then the comparation whit case "november", "december" ... The code looks great.

**Comment by xaca** [comment]
@sshirolkar Thank you, i made a suggestion. What do you thing?

**Comment by shivanishirolkar** [comment]
@xaca I'm sorry, but I can't seem to find the suggestion you made.

**Comment by sshirolkar** [comment]
@xaca You're right, that makes more sense.

File: SwitchStatements/src/season/Season.java (hunk 1 above)
Comment (shivanishirolkar) [reply SwitchStatements/src/season/Season.java:44]: You're right, that makes more sense.

**Pull request merged**